### ISSUE TITLE
How obtain more realistic image?

### ISSUE THREAD

**Issue by fabio-C** [NVIDIA/pix2pixHD#36]
Hi, can you suggest me how can I improve my results? Which train parameters should I change, and how?

These are my goals:

1 - I would like to obtain more realistic images.
2- I don't care about the similarity (difference) between the single output image and the relative goal image.

For example, in the DAY to NIGHT scene transformation, I don't care if the networks add a car that does't exist, or change the color of a house, or anything, but the result must seem realistic for an human eye.

**Comment by doantientai** [comment]
The only way is to try using different set of hyperparametter and see how it goes. Different data-sets require different hyperparametter-tunings. If you care more about the realism, you may try want to the pixel-wise loss and increase the Discriminator loss.

**Comment by fabio-C** [comment]
Thanks @doantientai. What do you mean with pixel-wise loss? Which is the options parameter to change in the code?

**Comment by doantientai** [comment]
Hi @fabio-C sorry I meant to _reduce_ the pixel-wise loss (the L1 loss between generated output and ground-truth) if they have, but it seems they don't use it in this paper. I think you may try adjusting the lambda in the equation 5 in the paper.
